.github/workflows/ci.yml

```diff
@@ -43,6 +42,19 @@ jobs:
         id: context
         uses: ./.github/actions/context
 
+  actionlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Run actionlint
+        run: |
+          docker run \
+            --rm \
+            -v "$(pwd):/workdir" \
+            -w "/workdir" \
+            rhysd/actionlint:latest \
```

Comment:
We should use a specific version w/ hash (and have it updated through dependabot later)